info/v2/conversion.go

```diff
@@ -96,7 +96,7 @@ func MachineStatsFromV1(cont *v1.ContainerInfo) []MachineStats {
 	return stats
 }
 
-func ContainerStatsFromV1(containerName string, spec *v1.ContainerSpec, stats []*v1.ContainerStats) []*ContainerStats {
+func ContainerStatsFromV1(containerName string, spec *v1.ContainerSpec, stats []*v1.ContainerStats, firstCall bool) []*ContainerStats {
```

Comment:
rather than `firstCall`, maybe just say `log bool` on whether it should log anything or not?  there are two log calls in this existing operation